backend/persistence/models/webhook.go

```diff
@@ -0,0 +1,44 @@
+package models
+
+import (
+	"github.com/gobuffalo/validate/v3/validators"
+	"time"
+
+	"github.com/gobuffalo/pop/v6"
+	"github.com/gobuffalo/validate/v3"
+	"github.com/gofrs/uuid"
+)
+
+// Webhook is used by pop to map your webhooks database table to your go code.
+type Webhook struct {
+	ID            uuid.UUID     `json:"id" db:"id"`
+	Callback      string        `json:"callback" db:"callback"`
+	Enabled       bool          `json:"enabled" db:"enabled"`
+	Failures      int           `json:"failures" db:"failures"`
+	ExpiresAt     time.Time     `json:"expires_at" db:"expires_at"`
```

Comment:
After reviewing the PR a second time, I was wondering why should a Webhook expire after some time? 
Because it is always frustrating, when you have a project which is not used much and after some time something does not work correct because there is something expired because of little usage.

Reply:
That was a security best practice I found. It prevents data leakage in case someone else buys the domain after an project was abandoned. I'm not hard set on keeping the expire mechanism. Alternatively we can introduce a config parameter to enable/disable the expiring mechanism.